cmd/mesh_bridge/Dockerfile

```diff
@@ -0,0 +1,29 @@
+# Note: this must be built from the root of the project with:
+#
+#     docker build . -f ./cmd/mesh_bridge/Dockerfile
+#
+
+# mesh-builder produces a statically linked binary
+FROM golang:1.13.4-alpine3.10 as mesh-builder
+
+
+RUN apk update && apk add ca-certificates nodejs-current npm make git dep gcc build-base musl linux-headers
+
+WORKDIR /0x-mesh
+
+ADD . ./
+
+RUN go build ./cmd/mesh_bridge
+
+# Final Image
+FROM alpine:3.9
```

Comment:
Should this be `alpine:3.9`? We are using `golang:1.13.4-alpine3.10` above.

Reply:
Okay, gotcha. I would think that we should update all 3, but maybe @albrow can comment on this?

Reply:
We should update them all, but it also likely doesn't matter much. We only switched to `alpine3.10` for the intermediate container that builds Mesh because `golang:1.13.4-alpine3.9` does not exist.

Reply:
Upgraded.